dll/win32/comdlg32/cdlg32.c

```diff
@@ -60,36 +56,37 @@ LPITEMIDLIST (WINAPI *COMDLG32_SHSimpleIDListFromPathAW)(LPCVOID) DECLSPEC_HIDDE
  *	FALSE if sibling could not be loaded or instantiated twice, TRUE
  *	otherwise.
  */
-static const char GPA_string[] = "Failed to get entry point %s for hinst = %p\n";
-#define GPA(dest, hinst, name) \
-	if(!(dest = (void*)GetProcAddress(hinst,name)))\
-	{ \
-	  ERR(GPA_string, debugstr_a(name), hinst); \
-	  return FALSE; \
-	}
-
 BOOL WINAPI DllMain(HINSTANCE hInstance, DWORD Reason, LPVOID Reserved)
 {
-	TRACE("(%p, %d, %p)\n", hInstance, Reason, Reserved);
+	TRACE("(%p, %ld, %p)\n", hInstance, Reason, Reserved);
 
 	switch(Reason)
 	{
 	case DLL_PROCESS_ATTACH:
+	{
+		ACTCTXW actctx = {0};
+
 		COMDLG32_hInstance = hInstance;
 		DisableThreadLibraryCalls(hInstance);
-
-		SHELL32_hInstance = GetModuleHandleA("SHELL32.DLL");
 #ifdef __REACTOS__
-		InitializeCriticalSection(&COMDLG32_OpenFileLock);
+		InitializeCriticalSection(&COMDLG32_OpenFileLock);  // Is this still needed?
```

Comment:
Yes, I do think that is still needed for the Shell Change Notification which was implemented during 0.4.14-dev'ing by Katayama for the file-browser-dlg.

Reply:
Noted, that function was changed a good amount so I wasn't sure whether it still applies